hermes/file_gen_test.go

```diff
@@ -0,0 +1,39 @@
+package hermes
+
+import(
+	//"fmt"
+	"testing"
+	"math/rand"
+	"strconv"
+	"time"
+	//"crypto/sha1"
+)
+
+
+func TestChecksum(t *testing.T){
+	var global_contents string = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+	var result_expected string = "_68f3caf439065824dcf75651c202e9f7c28ebf07"
+	file := HermesFile{}
+	file.contents = global_contents;
+	result := generate_file_checksum(file)
+	if result != result_expected {
+		t.Errorf("generate_file_checksum(\"jhfvjhdfjhfjjhjhdfvjvcvfjh\") failed expected %v got %v", result_expected, result)
+	}
+}
+
+func TestFileName(t *testing.T){
+	rand.Seed(time.Now().UnixNano())
+	var file_ID int = rand.Intn(40)+10;
```

Comment:
- As mentioned above, use := when declaring and assigning in the same line. Ditto for the assignments below. Alternatively, you can assign them as a block: https://golang.org/doc/effective_go.html#variables.
- You don't need a semi-colon (https://golang.org/doc/effective_go.html#semicolons)

```suggestion
	file_ID := rand.Intn(40)+10
```